app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -115,9 +125,14 @@ class BloodPressureHistoryScreen(
   }
 
   private fun setupBloodPressureHistoryList() {
+    val dividerMarginStart = 8.dp
+    val dividerMarginEnd = 8.dp
```

Comment:
Both these values are same so no need to create two separate variables.
Just create a variable like `dividerMargin`.